conf.yaml

```diff
@@ -291,20 +291,20 @@ contents:
             chunk:      1
             tags:       XPack/Reference
             current:    5.3
-            index:      docs/public/index.asciidoc
+            index:      docs/en/index.asciidoc
             private:    1
             branches:   [ 5.3, 5.2, 5.1, 5.0 ]
             sources:
               -
                 repo:   x-pack
-                path:   docs/public
+                path:   docs/en
               -
                 repo:   x-pack-kibana
-                path:   docs/public-kb
```

Comment:
It's not used by 5.3 or earlier.  But in 5.x and master, it *is* used, and it *is* renamed.